src/render/SDL_render.c

```diff
@@ -4493,8 +4493,12 @@ int SDL_SetRenderVSync(SDL_Renderer *renderer, int vsync)
     return 0;
 }
 
-SDL_bool SDL_GetRenderVSync(SDL_Renderer *renderer)
+int SDL_GetRenderVSync(SDL_Renderer *renderer, int *vsync)
 {
-    CHECK_RENDERER_MAGIC(renderer, SDL_FALSE);
-    return renderer->wanted_vsync;
+    CHECK_RENDERER_MAGIC(renderer, -1);
+    if (vsync == NULL) {
+        SDL_InvalidParamError("vsync");
```

Comment:
This should be `return SDL_Invalid...`

Reply:
Or maybe just only fill it in if it's non-NULL but still return 0. I think we do that in some other functions as well.